src/applications/mhv-medical-records/reducers/labsAndTests.js

```diff
@@ -38,11 +39,20 @@ const initialState = {
   labsAndTestsDetails: undefined,
 };
 
+const getLabLocation = (performer, record) => {
+  if (isArrayAndHasItems(performer)) {
+    const locationRef = performer[0]?.reference;
+    const labLocation = extractContainedResource(record, locationRef);
+    return labLocation?.name;
+  }
+  return null;
+};
+
 /**
  * @param {Object} record - A FHIR chem/hem Observation object
  * @returns the appropriate frontend object for display
  */
-const convertChemHemObservation = results => {
+const convertChemHemObservation = (results, record) => {
```

Comment:
The results are contained within the record. I would just pass record and pull results out here if this is the only place we need them.

Reply:
Good point!